openwisp_users/api/serializers.py

```diff
@@ -19,6 +19,12 @@
 
 
 class OrganizationSerializer(ValidatedModelSerializer):
+    slug = serializers.SlugField(
+        allow_unicode=False,
+        help_text=_('The name in all lowercase, suitable for URL identification'),
+        max_length=200,
+    )
```

Comment:
Because this field already has `validators = [UniqueValidator(queryset=Organization.objects.all())>`, I redefined it to use a customised validation error message response. If I recall correctly, and after some thought, I discovered it can avoid this by removing validations by [extra_kwargs (DRF doc)](https://www.django-rest-framework.org/api-guide/serializers/#additional-keyword-arguments) in the `meta` class attribute; see below: -

```py
		# we can use this instead of the redefining serializer field above
        extra_kwargs = {
            'slug': {
                'validators': []
            }
        }
```

Reply:
![Screenshot from 2023-01-04 16-27-14](https://user-images.githubusercontent.com/56113566/210543018-f2610b1c-082d-4cbd-9b7d-6e7b12653370.png)

Reply:
Added in [`12a1435` (#334)](https://github.com/openwisp/openwisp-users/pull/334/commits/12a1435b45e771f9f9fd19337a5757729f800a1d)